client/core/bond.go

```diff
@@ -713,8 +713,6 @@ func (c *Core) rotateBonds(ctx context.Context) {
 		}
```

Comment:
I think we have a problem above here because if the dex doesn't exist any more, we'll never get a config, and we'll never get to `refundExpiredBonds`.

Reply:
Also take a look at the `categorizeBonds` inline function. Does that only run after we connect? Does that mean that we have to connect to the server in order to do refunds?

If so, I think that's a holdover from when we didn't know bond strength and had to base it on the dex configuration, something that was resolved with v1.